[Metrics Advisor] update changelog (#13593)
Azure/azure-sdk-for-js: sdk/metricsadvisor/ai-metrics-advisor/CHANGELOG.md

## 1.0.0-beta.3 (Unreleased)

- Added AAD authentication support
- Added support for API key and Subscription key rotation
- Added a map property in DataFeed object for mapping of metric name to metric id

> - [Breaking] Extra GET calls removed from all `Update` methods, as a result the methods now return just RestResponse instead of entire objects:
>     - `updateAlertConfig`
>     - `updateDatafeed`
>     - `updateHook`
>     - `updateDetectionConfig`

Reviewer: Removing extra GET call is an implementation details that our consumers don't care about. Better to have more info about API changes
```suggestion
- [Breaking] All update methods now return just RestResponse instead of entire objects:
    - `updateAlertConfig()` now returns `RestResponse` instead of `<insert the old return type>`
    - `updateDatafeed` <similar>
    - `updateHook` <similar>
    - `updateDetectionConfig` <similar>
```
